common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerOperationExecutor.java

```diff
@@ -109,13 +112,20 @@ public interface BrokerOperation<T> {
         void putValueInSuccessEvent(@NonNull final ApiEndEvent event, @NonNull final T result);
     }
 
+    private final ActiveBrokerCacheUpdaterUtil mCacheUpdaterUtil;
+
     private final List<IIpcStrategy> mStrategies;
+    private final IActiveBrokerCache mActiveBrokerCache;
 
     /**
      * @param strategies list of IIpcStrategy to be invoked.
      */
-    public BrokerOperationExecutor(final @NonNull List<IIpcStrategy> strategies) {
+    public BrokerOperationExecutor(@NonNull final List<IIpcStrategy> strategies,
+                                   @NonNull final IActiveBrokerCache activeBrokerCache,
+                                   @NonNull final ActiveBrokerCacheUpdaterUtil cacheUpdaterUtil) {
         mStrategies = strategies;
+        mActiveBrokerCache = activeBrokerCache;
+        mCacheUpdaterUtil = cacheUpdaterUtil;
```

Comment:
Why do you choose to provide the Util as a constructor parameter instead of just invoking the Util class?

Reply:
I assume this is done for testing purposes.